packages/jellyfish-api-core/__tests__/category/loan/destroyLoanScheme.test.ts

```diff
@@ -0,0 +1,163 @@
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+import { UTXO } from '@defichain/jellyfish-api-core/category/loan'
+import BigNumber from 'bignumber.js'
+import { Testing } from '@defichain/jellyfish-testing'
+
+describe('Loan', () => {
+  const container = new LoanMasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+
+  beforeAll(async () => {
+    await testing.container.start()
+    await testing.container.waitForWalletCoinbaseMaturity()
+
+    // NOTE(jingyi2811): Default scheme
```

Comment:
`NOTE(name)` is quite abuse for me.. the use case of *NOTE* should be something require us to NOTE
```suggestion
    // Default scheme
```

Reply:
i add on the practice of **NOTE**
the use case:
i . abnormal approach: note the reason to reader why doing the logic on this way
ii. reference: something is required to understand before read this
...